ColorPair_test_data.py

```diff
@@ -0,0 +1,19 @@
+from ColorPair_get_data import get_color_from_pair_number
+from ColorPair_get_data import get_pair_number_from_color
+
+def test_functionalities():
+  test_number_to_pair(4, 'White', 'Brown')
+  test_number_to_pair(5, 'White', 'Slate')
+  test_pair_to_number('Black', 'Orange', 12)
+  test_pair_to_number('Violet', 'Slate', 25)
+  test_pair_to_number('Red', 'Orange', 7)
+   
+def test_number_to_pair(pair_number,
+                        expected_major_color, expected_minor_color):
```

Comment:
Good practice to separate tests into a file!
Try renaming the test functions to include the expectations. Do new tests emerge?